spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/bridge/BraveHttpServerRequest.java

```diff
@@ -131,6 +133,10 @@ private boolean resolveFromInetAddress(brave.Span span) {
 					}
 					return span.remoteIpAndPort(addr.getAddress().getHostAddress(), addr.getPort());
 				}
+				else if (delegate instanceof HttpServletRequest) {
```

Comment:
I'm thinking that this might be just `ServletRequest`.

Reply:
Done